src/tests/system/tests/test_netgroups.py

```diff
@@ -108,3 +124,282 @@ def test_netgroups__add_remove_netgroup_member(client: Client, provider: Generic
+        3. Add another member to the "QAUsers" netgroup.
+        4. Check if after the cache expires, the user ng9001 and host testhost2
+            are now correctly considered part of the netgroup "QAUsers".
+        5. Check if both tuples (testhost1, ng9000, ldap.test) and (testhost2, ng9001, ldap.test)
+            are present in the "QAUsers" netgroup after the changes.
+    :expectedresults:
+        1. Netgroup "QAUsers" contains the tuple (testhost1, ng9000, ldap.test)
+        2. Tuple (testhost1, ng9001, ldap.test) does not exist in the netgroup
+        3. Another member should be added to the "QAUsers" netgroup.
+        4. The user ng9001 and host testhost2 are now correctly considered part of the netgroup "QAUsers".
+        5. Both tuples (testhost1, ng9000, ldap.test) and (testhost2, ng9001, ldap.test)
+            are present in the "QAUsers" netgroup after the changes.
+    :customerscenario: False
+    """
+    ou = ldap.ou("Netgroup")
+    create_users(ldap)
+
+    qa_users = ldap.netgroup("QAUsers", basedn=ou).add()
+    qa_users.add_member(host="testhost1", user="ng9000", domain="ldap.test")
+
+    client.sssd.dom("test")["entry_cache_timeout"] = "60"
+    client.sssd.start()
+
+    assert "(testhost1, ng9000, ldap.test)" in client.tools.getent.netgroup("QAUsers").members
+    assert not "(testhost1, ng9001, ldap.test)" in client.tools.getent.netgroup("QAUsers").members
+
+    qa_users.add_member(host="testhost2", user="ng9001", domain="ldap.test")
+
+    time.sleep(70)
+
+    assert "(testhost1, ng9000, ldap.test)" in client.tools.getent.netgroup("QAUsers").members
+    assert "(testhost2, ng9001, ldap.test)" in client.tools.getent.netgroup("QAUsers").members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_netgroup__multiple_netgroups(client: Client, ldap: LDAP):
+    """
+    :title: Create multiple netgroups
+    :setup:
+        1. Create users, groups and start sssd.
+    :steps:
+        1. Verify if the tuple (testhost5,ng9005,ldap.test) exists in the "DEVUsers" netgroup,
+            using the getent netgroup command.
+        2. Check if the combination of host (testhost5), user (ng9005), and domain (ldap.test)
+            is correctly part of the "DEVUsers" netgroup using libc.innetgr.
+    :expectedresults:
+        1. Tuple (testhost5,ng9005,ldap.test) exists in the "DEVUsers" netgroup
+        2. Combination of host (testhost5), user (ng9005), and domain (ldap.test)
+            is correctly part of the "DEVUsers" netgroup
+    :customerscenario: False
+    """
+    ou = ldap.ou("Netgroup")
+    create_users(ldap)
+
+    qa_users = ldap.netgroup("QAUsers", basedn=ou).add()
+    qa_users.add_member(host="testhost1", user="ng9000", domain="ldap.test")
+    dev_users = ldap.netgroup("DEVUsers", basedn=ou).add()
+    dev_users.add_member(host="testhost5", user="ng9005", domain="ldap.test")
+
+    client.sssd.dom("test")["entry_cache_timeout"] = "60"
```

Comment:
This doesn't need to be set, let's only update the configuration if the test needs it, otherwise let's keep the default configuration.